javascript/javascript-4.md

```diff
@@ -88,3 +89,8 @@ The output should look similar to this:
 `Erin is willing to go to Banff, Alberta, Canada or Iceland.`  
 `Janet is willing to go to London or Hogwarts.`  
 
+### Check for Understanding
+
+- What are the different types of loops in JavaScript?
+- Differentiate between for loop and while loop?
```

Comment:
You can also mention do-while loop.

Reply:
Kundan, is do-while covered in this lesson? No new info should be covered in check for understanding.

Reply:
Also, line 95 is not a question @raibove

Reply:
But then the meaning of the first question goes wrong...as there are 3 types of loops in JavaScript...

Reply:
Can you check if do-while is in the lesson?   If not, she should rephrase line 94.  If it is, she should follow your first comment here.

Reply:
I have checked, do-while loop is not there in the lesson.

Reply:
@alodahl  @Kundan28  I have changed the questions Please review the new questions.